src/react/components/pages/predict/predictPage.tsx

```diff
@@ -250,24 +250,33 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
                                                 <h5 className="model-selection-header">
                                                     {strings.predict.selectModelHeader}
                                                 </h5>
-                                                <h6 className="model-selection-info-header" >
-                                                    <span className="model-selection-info-key">
-                                                        {strings.predict.modelIDPrefix}
-                                                    </span>
-                                                    <span className="model-selection-info-value">
-                                                        {mostRecentModel.modelInfo.modelId.substring(0,8) + "..."}
-                                                    </span>
-                                                </h6>
-                                                {mostRecentModel.modelInfo.modelName &&
+                                                <tr>
                                                     <h6 className="model-selection-info-header" >
                                                         <span className="model-selection-info-key">
-                                                            {strings.predict.modelNamePrefix}
+                                                            {strings.predict.modelIDPrefix}
                                                         </span>
                                                         <span className="model-selection-info-value">
-                                                            {mostRecentModel.modelInfo.modelName}
+                                                            {mostRecentModel.modelInfo.modelId.substring(0,8) + "..."}
```

Comment:
can we use CSS to achieve this?

Reply:
Done
![image](https://user-images.githubusercontent.com/60453211/87969461-0ca0b880-ca77-11ea-8ed5-5953b608b58e.png)